src/lib/storage/table.cpp

```diff
@@ -200,7 +205,10 @@ void Table::append_chunk(const Segments& segments, std::shared_ptr<MvccData> mvc
   }
 #endif
 
-  _chunks.push_back(std::make_shared<Chunk>(segments, mvcc_data, alloc));
+  auto c = std::make_shared<Chunk>(segments, mvcc_data, alloc);
```

Comment:
I wish we could teach clang-tidy to prohibit single character identifiers

Reply:
Right. I think I got inspired by one of the test cases.